deblur/deblurring.py

```diff
@@ -55,7 +55,34 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
+def create_error_profile(read_error):
+    """Create a theoretical read error profile given
+    a maxima error rate
```

Comment:
it is a comma separated list of upper bound on error probability per hamming distance.
let's denote p(d) the upper bound on the probability to get d changes in the sequence (hamming distance). the error probability is [p(0),p(1),p(2),p(3),...p(n)].
while ideally p(2) should be p(1)^2, it is not.
n is determined by when the error probability gets so small it's practically 0 (we use n=11, but p(10) is set to very small so not critical - p is decreasing fast).
the error profile is based on phix reads, mock mixtures and samples with known bacteria.